src/components/views/dialogs/spotlight/SpotlightDialog.tsx

```diff
@@ -408,7 +408,7 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", initialFilter = n
                         !entry.query?.some(q => q.includes(lcQuery))
                     ) return; // bail, does not match query
                 } else if (isMemberResult(entry)) {
-                    if (!entry.query?.some(q => q.includes(lcQuery))) return; // bail, does not match query
+                    // Do not filter users
```

Comment:
This comment needs to say *WHY* (because the rely on the server to filter *DIRECTORY* results for us)

Reply:
Can you elaborate why clientside filtering is done in the first place? The spec suggests nothing of the kind, and I think additional validation seems superfluous since the server should be trustworthy enough.

Reply:
Probably a copy-pasta, but you'd need to ask whomever wrote it